subprojects/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/JacocoIntegrationTest.kt

```diff
@@ -14,6 +16,7 @@ class JacocoIntegrationTest : AbstractPluginIntegrationTest() {
     @ToBeFixedForInstantExecution
     fun `jacoco ignore codegen`() {
 
+        assumeTrue(TestPrecondition.JDK14_OR_EARLIER.isFulfilled) // reevaluate when upgrading JaCoco from current 0.8.5
```

Comment:
`@Require(TestPrecondition.JDK14_OR_EARLIER)` does not do the trick in this module and is ignored.